ros/src/fsds_ros_bridge/launch/fsds_ros_bridge.launch

```diff
@@ -7,6 +7,7 @@
 	<arg name="manual_mode" default="false"/>
 	<arg name="plot" default="false"/>
 	<arg name="rviz" default="false"/>
+	<arg name="UDP_control" default="false"/>
```

Comment:
Would you like to add this argument in the arguments table in the `ros-bridge.md` docs file?